### PR TITLE
Mango bug

### PR DESCRIPTION
**Pull request by ChristineWangcy**
## Description

Fixes # (issue)
intermittent bug in connection script for Mongo

## Type of change
import certifi module
add tlsCAFile=certifi.where() in MongoClient:
        return MongoClient(getenv("MONGO_URL"), tlsCAFile=certifi.where())[self.cluster][collection]

Please delete options that are not relevant.

- [x] Bug fix

## Checklist:

- [x] My code follows the PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by rodricobsanchez** [comment]
Looks really good!

**Comment by Annapurnaj91** [comment]
It would be good if u describe more about the type of bug changes you made. Other than that everything looks good.

**Review by RylandGomez — CHANGES_REQUESTED** [review]
The comments on line 28 seem more like a reminder for yourself and not an explanation of the code. The commented out code on line 27 should also be removed. Finally, the empty line 17, in my opinion, could be removed, since certifi is being used in conjunction with the pymongo import (I like to keep imports with similar purposes together).

Functionally, the code is great and helps ensure successful connection to the database! Once the code gets cleaned up a little, I'd happily approve!

File: app/data.py [hunk 1]
```diff
@@ -58,4 +57,4 @@ def drop_index(self, collection: str):
         self._connect(collection).drop_index([("$**", "text")])
 
     def search(self, collection: str, user_search: str) -> List[Dict]:
-        return self.read(collection, {"$text": {"$search": user_search}})
+        return self.read(collection, {"$text": {"$search": user_search}})
```
Comment (RylandGomez) [inline app/data.py:60]: Add an empty line (61) at the end of this for PEP8

**Review by RylandGomez — CHANGES_REQUESTED** [review]
PEP8 issue needs to be resolved at the bottom of app.py. I still think the certifi comment on line 27 seems unnecessary, but if you have something you want explained there and need help defining it, I'm happy to help!

File: app/data.py [hunk 2]
```diff
@@ -24,7 +24,8 @@ def __init__(self, cluster: str):
         self.cluster = cluster
 
     def _connect(self, collection: str) -> MongoClient:
-        return MongoClient(getenv("MONGO_URL"))[self.cluster][collection]
+        # add certifi
```
[the remark this answers is not in the document]
Comment (RylandGomez) [reply app/data.py:27]: Something like:

# certifi added to resolve ___

might be useful to quickly answer questions future devs have as to why it's there without them having to do too much research.

**Comment by ulloa** [comment]
Hi, what is the issue with mongo we are having right now?

**Comment by ChristineWangcy** [comment]
Hi,

I requested a new PR based on Ryan's suggestion and added detailed
information about the issue in the PR. Thanks for the reviews and comments
from Rodrico, Annapurna and Ryland.

**Pull request closed without merging**